scripts/linux/pkg/deb/assets/ravendb/debian/ravendb.postinst

```diff
@@ -4,9 +4,17 @@ set -e
 
 case "$1" in
     configure)
-        
-        adduser --disabled-login --disabled-password --system --home /var/empty --no-create-home --quiet --group ravendb
+        useradd_uid=""
+        useradd_gid=""
+        if [ -n "$RAVEN_UserID" ]; then
+            useradd_uid="--uid $RAVEN_UserID"
+        fi
 
+        if [ -n "$RAVEN_GroupID" ]; then
+            useradd_gid="--gid $RAVEN_GroupID"
+        fi
```

Comment:
Also let's use uppercase letters for env vars `RAVEN_USER_ID` and `RAVEN_GROUP_ID` to differentiate from env vars setting RavenDB server settings.

```suggestion
[ -n "$RAVEN_UserID" ] && useradd_uid="--uid $RAVEN_UserID"
[ -n "$RAVEN_GroupID" ] && useradd_gid="--gid $RAVEN_GroupID"
```